components/brave_today/browser/publishers_parsing.cc

```diff
@@ -71,6 +72,27 @@ bool ParseCombinedPublisherList(const std::string& json,
       continue;
     }
     publisher->site_url = site_url;
+
+    auto* favicon_url_raw = publisher_raw.GetDict().FindString("favicon_url");
```

Comment:
Should we save `publisher_raw.GetDict()` to a local variable?

Reply:
Yeah, it was on my list of things to do. Basically, the old API was deprecated, but I didn't want to update the whole parse method to use the new API in the PR. I don't think it makes much of a difference though so I've converted the whole function.